FlymFork/src/main/res/values-de/strings.xml

```diff
@@ -1,8 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!--
+    <!--
+ Copyright (c) 2012-2015 Frederic Julian
  Copyright (c) 2010-2012 Stefan Handschuh
- Copyright (c) 2012-2014 Frederic Julian
-      modified 2014-2016 Matthias Dill
```

Comment:
Why do you want to remove this copyright?

Reply:
I do not want. This happens at Oneskyapp. You can add it again.

Reply:
I added it...